byzcoin/ringbuf.go

```diff
@@ -0,0 +1,44 @@
+package byzcoin
+
+import (
+	"sync"
+)
+
+func newRingBuf(size int) ringBuf {
+	return ringBuf{
+		current: 0,
+		size:    size,
+		items:   make([]ringBufElem, size),
+	}
+}
+
+type ringBufElem struct {
+	key   string
```

Comment:
why is this `string` and not `[]byte`?

Reply:
because history... it started as a map then changed to a slice but the string type stayed

now i've changed it to `[]byte`